From jkosina@suse.cz  Thu Dec  7 09:58:11 2006
Date: Thu, 7 Dec 2006 17:58:40 +0100 (CET)
From: Jiri Kosina <jkosina@suse.cz>
To: Greg KH <greg@kroah.com>, Marcel Holtmann <marcel@holtmann.org>
Cc: Dmitry Torokhov <dmitry.torokhov@gmail.com>
Subject: Generic HID layer - disable USB HID
Message-ID: <Pine.LNX.4.64.0612071747380.4215@jikos.suse.cz>
MIME-Version: 1.0
Content-Type: TEXT/PLAIN; charset=US-ASCII

[PATCH] Generic HID layer - disable USB HID

This patch is a part of generic HID layer introduction. USB HID
is disabled, so that the code split and changes could be introduced
in a way that is reviewable (i.e. separate patches), but not to break
git bisect by uncompilable kernel throughout different stages of the
code splitup and changes. The last patch of this series enables HID
again.

Signed-off-by: Jiri Kosina <jkosina@suse.cz>
Signed-off-by: Marcel Holtmann <marcel@holtmann.org>
Cc: Dmitry Torokhov <dmitry.torokhov@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/input/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/input/Kconfig
+++ gregkh-2.6/drivers/usb/input/Kconfig
@@ -6,7 +6,7 @@ comment "USB Input Devices"
 
 config USB_HID
 	tristate "USB Human Interface Device (full HID) support"
-	depends on USB
+	depends on USB && BROKEN
 	---help---
 	  Say Y here if you want full HID support to connect keyboards,
 	  mice, joysticks, graphic tablets, or any other HID based devices
